airflow/providers/amazon/aws/auth_manager/avp/entities.py

```diff
@@ -35,8 +35,10 @@ class AvpEntities(Enum):
     # Resource types
     CONFIGURATION = "Configuration"
     CONNECTION = "Connection"
+    CUSTOM = "Custom"
     DAG = "Dag"
     DATASET = "Dataset"
+    MENU = "Menu"
```

Comment:
Those changes for MENU look unrelated. Or am I wrong?

Reply:
See my comment [here](https://github.com/apache/airflow/pull/37915#issuecomment-1985892661)

Reply:
Fine for me